azure_functions_worker/testutils.py

```diff
@@ -71,26 +71,39 @@
 HOST_JSON_TEMPLATE = """\
 {
     "version": "2.0",
-    "logging": {
-        "logLevel": {
-           "default": "Trace"
-        }
-    },
-    "http": {
-        "routePrefix": "api"
-    },
-    "swagger": {
-        "enabled": true
-    },
-    "eventHub": {
-        "maxBatchSize": 1000,
-        "prefetchCount": 1000,
-        "batchCheckpointFrequency": 1
-    },
-    "functionTimeout": "00:05:00"
+    "logging": {"logLevel": {"default": "Trace"}}
 }
 """
 
+EXTENSION_CSPROJ_TEMPLATE = """\
```

Comment:
Adding a new csproj file. V3 didn't add a csproj file at the top and only placed a `bin` folder during the testing. This didn't work to load the library during v4 tests.